lambda/apiprocessor/search.py

```diff
@@ -84,54 +58,38 @@ def search(request):
         awsauth = AWS4Auth(credentials.access_key, credentials.secret_key,
                            region, service, session_token=credentials.token)
 
-        print(searchBody)
-
         es = Elasticsearch(
             hosts=[{'host': host, 'port': 443}],
             http_auth=awsauth,
             use_ssl=True,
             verify_certs=True,
             connection_class=RequestsHttpConnection
         )
+        
         output = es.search(
             index='textract',
             doc_type="document",
-            body={
-                "query": {
-                  "bool": searchBody
-                }
-            },
-            filter_path=['hits.hits._id', 'hits.hits._source.content',
```

Comment:
Is this filtering no longer necessary?

Reply:
Good catch. I have added it back with an additional source filter.